upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.6.yaml.template

```diff
@@ -119,7 +119,7 @@ spec:
                   fieldPath: spec.nodeName
             # Set Felix endpoint to host default action to ACCEPT.
             - name: FELIX_DEFAULTENDPOINTTOHOSTACTION
-              value: "ACCEPT"
+              value: "{{- if eq .Networking.Canal.DefaultEndpointToHostAction "" }}ACCEPT{{- else -}}{{ .Networking.Canal.DefaultEndpointToHostAction }}{{- end -}}"
```

Comment:
I _think_ you can also use `{{- or  .Networking.Canal.DefaultEndpointToHostAction "ACCEPT" }}` but this works :-)

Reply:
Yep that works and cleaner, will update. :) Cheers!